search-app/test/cmr/search/test/unit/services/acls/granule_acls.clj

```diff
@@ -91,7 +91,7 @@
         old-mem-cache (get-in context [:system :caches :collections-for-gran-acls])
         context (assoc-in context [:system :caches :collections-for-gran-acls] rcache)
         data (.get-value old-mem-cache :collections)]
-    (.set-value rcache :collections-for-gran-acls data)
+    (.set-values rcache :collections-for-gran-acls data)
```

Comment:
Why did you leave old-mem-cache in the test?  Can you remove this and just set up the redis cache as the old-mem-cache (replace old-mem-cache)?  All you do with it is setup the memory cache and then take its contents and put it into the redis cache and then not use it anymore.

Reply:
I don't think I even need this code, I removed it and tests pass locally, testing on bamboo now.

Reply:
switched to the clojure syntax as suggested.